R/method-differentialAbundance.R

```diff
@@ -238,13 +239,13 @@ setMethod("maaslin", signature("AbundanceData", "Comparator"), function(data, co
 #' @useDynLib microbiomeComputations
 #' @export
 setGeneric("differentialAbundance",
-  function(data, comparator, method = c('DESeq', 'Maaslin'), verbose = c(TRUE, FALSE)) standardGeneric("differentialAbundance"),
+  function(data, comparator, method = c('DESeq', 'Maaslin'), usePValueFloor = c(TRUE, FALSE), verbose = c(TRUE, FALSE)) standardGeneric("differentialAbundance"),
```

Comment:
At first this made a lot of sense, but after some more thought i realized that means it's the frontend that has to keep this value consistent across our apps. That seems like a job that's better suited for this package instead of the frontend. And then if we return the pvalue floor that we use, like you suggested, the frontend can still be kept in sync because it just reads the floor and then doesn't have to worry about staying consistent across frontend plugins

Reply:
Does it have to be consistent across our apps? It seems a flaw in the API that you can floor the value but not dictate what the floor should be. I mean, is it possible different contexts will want different floors? We floored p values in genomics once that I remember at .001, bc in a different context we had different reqs. We want the floor real low here to maintain granularity in the volcano plot.. but outside a volcano plot idk.